src/service/member-api-adapter.js

```diff
@@ -2,11 +2,13 @@
  *
  * ##Member API Adapter
  *
- * The Member API Adapter provides methods to lookup user details or group details
+ * The Member API Adapter provides methods to look up information about end users for your project and how they are divided into groups. It is based on query capabilities of the underlying RESTful [Member API](../../../rest_apis/user_management/member/).
  *
- *      var auth = new F.adapter.Member();
- *      auth.getGroupsByUser({ userId: 'b6b313a3-ab84-479c-baea-206f6bff337' });
- *      auth.getGroupDetails({ groupId: '00b53308-9833-47f2-b21e-1278c07d53b8' });
+ * Typically this is only needed for Authenticated projects, that is, team projects with [end users and groups](../../../groups_and_end_users/). For example, if some of your end users are facilitators, or if your end users should be treated differently based on which group they are in, use the Member API to find that information.
+ *
+ *      var ma = new F.service.Member({ token: 'user-or-project-access-token' });
```

Comment:
it doesn't seem to. i filed https://github.com/forio/epicenter-js-libs/issues/110, but when that's fixed i'll remove from here.